### ISSUE TITLE
[tr064] Binding OH3 MAC adresses for LANDevice

### ISSUE THREAD

**Issue by FrankH1** [openhab/openhab-addons#13798]
Did install the TR-064 Binding - just with the purpose of detecting the "onlinestatus" of mobiles in the household. Went up to the subdevice (LANDevice). Device and Subdevice (FritzBox) both are online. Need to add I'm doing this via the GUI of OH3. If you start to add several  MAC adresses, separated by "," as requested by description then only for the last adress the system offers channels like MAConline etc. Did discuss this in the german OPENHAB-Forum. The souliution was to skip the commas - so the adresses are filled in the the GUI in separate lines - but without "," or other separators. This works and I'm able to get the status of connected devices.

So either the Addin needs to be changed to work with "," as separator - or the documentation needs update/change.

## Your Environment

* openHAB 3.3.0 Release Build, TR-064 3.3.0
* Raspbian, Raspi 4B

**Comment by lsiepel** [comment]
Can you point me to that discussion ? It looks like it is a documenation problem. I also see a difference in the english and german translation.

I assume you tried this and it failed:
```
Thing subdeviceLan LAN "label LAN"   [ uuid="uuid:xxxxxxxx-xxxx-xxxx-yyyy-xxxxxxxxxxxx",  macOnline="XX:XX:XX:XX:XX:XX", "YY:YY:YY:YY:YY:YY"]
```
                                                          
You then tried this and that succeedded:
```
Thing subdeviceLan LAN "label LAN"   [ uuid="uuid:xxxxxxxx-xxxx-xxxx-yyyy-xxxxxxxxxxxx", macOnline="XX:XX:XX:XX:XX:XX" "YY:YY:YY:YY:YY:YY"]  
```

**Comment by FrankH1** [comment]
Hi, 
I did everything on the gui. In the gui there is a box, where you can put in MAC addresses, and there is an explaining text below, saying adresses need to be separated by commas. ( Can't check as I'm now on holidays far away from cold Germany...)
The original topic where I raised that is in the German forum: https://openhabforum.de/viewtopic.php?t=7578
In the post from sihui is a screenshot, where the corresponding part of the gui is shown. Hope that helps. Cheers

**Comment by jlaur** [comment]
Related PR merged: #13990
